### ISSUE TITLE
Linkedin Oauth fails of user is missing a profile picture

### ISSUE THREAD

**Issue by aoatridge** [laravel/socialite#328]
Using
`"laravel/socialite": "^4.0"` 

I get a "array_filter() expects parameter 1 to be array, null given" when it tries to run mapUserToObject() in socialite/src/Two/LinkedInProvider.php.

My test account is missing an avatar image. 
var_dump of the $user object passed to it.
```
array (size=4)
  'firstName' => 
    array (size=2)
      'localized' => 
        array (size=1)
          'en_US' => string 'Mark' (length=4)
      'preferredLocale' => 
        array (size=2)
          'country' => string 'US' (length=2)
          'language' => string 'en' (length=2)
  'lastName' => 
    array (size=2)
      'localized' => 
        array (size=1)
          'en_US' => string 'Smith' (length=8)
      'preferredLocale' => 
        array (size=2)
          'country' => string 'US' (length=2)
          'language' => string 'en' (length=2)
  'id' => string 'BhcxETf1d3' (length=10)
  'emailAddress' => string 'myemail@email.net' (length=24)
```

**Comment by aoatridge** [comment]
Confirmed that it works with a linkedin account that has an avatar pic.

**Comment by Crypton33** [comment]
I'm experiencing a similar problem, but in my case even this code doesn't work:
`Socialite::driver('linkedin')->user()`

I'm getting the error:
`array_filter() expects parameter 1 to be array, null given`
And it happens in the same function.

**Comment by driesvints** [comment]
@Crypton33 probably the same error. And it can't have happened on that line because there's no array_filter call there. It's probably happening inside the `mapUserToObject` call.

**Comment by driesvints** [comment]
Released 4.0.2 with a bugfix for this. Can you try again?

**Comment by Crypton33** [comment]
Works fine now, thanks 👍
